src/cocoa/toga_cocoa/widgets/scrollcontainer.py

```diff
@@ -51,3 +51,26 @@ def set_horizontal(self, value):
     def rehint(self):
         self.interface.intrinsic.width = at_least(self.interface.MIN_WIDTH)
         self.interface.intrinsic.height = at_least(self.interface.MIN_HEIGHT)
+
+    def set_on_scroll(self, on_scroll):
+        self.interface.factory.not_implemented("ScrollContainer.set_on_scroll()")
+
+    def get_vertical_position(self):
+        self.interface.factory.not_implemented(
+            "ScrollContainer.get_vertical_position()"
+        )
```

Comment:
The default return value for this method will be `None`, which won't be a valid value ordinarily. I'd suggest we should be returning a dummy value (e.g., 0) that is at least type compatible with the expected values. That way, the failures will be limited to "wrong position", not "type errors because of unexpected return types".